src/RageSoundPosMap.h

```diff
@@ -14,8 +14,12 @@ class pos_map_queue
 	pos_map_queue( const pos_map_queue &cpy );
 	pos_map_queue &operator=( const pos_map_queue &rhs );
 
-	/* Insert a mapping from iSourceFrame to iDestFrame, containing iFrames. */
-	void Insert( std::int64_t iSourceFrame, int iFrames, std::int64_t iDestFrame, float fSourceToDestRatio = 1.0f );
+	/* Insert a mapping from iSourceFrame to iDestFrame, containing iFrames.
+	*
+	* To maintain good precision while increasing iFrames to int64_t, fSourceToDestRatio should
```

Comment:
I don't think we have to mention converting the float to a double, and reword to emphasize we use doubles to maintain precision to prevent any sync drift issues down the line.